src/actions/index.js

```diff
@@ -1,5 +1,11 @@
 import localforage from 'localforage';
 import Client from '../client';
+import moment from 'moment';
+
+export const CART_ITEM_KEY = 'cartItems';
+export const CART_ADDRESS_KEY = 'cartAddress';
+export const DELIVERY_DATE_KEY = 'deliveryDate';
+export const ORDER_KEY = 'order';
 
 export const addToCart = (menuItem, modifiers = {}) => {
```

Comment:
do you know property value shorthand in es6 ?
(http://www.benmvp.com/learning-es6-enhanced-object-literals/)
it makes the syntax even shorter now:)

for example here you could rewrite like:
```

```

Reply:
Yeah I am an old jQuery guy you know..